### DISCUSSION TITLE
Add child note from side note

### DISCUSSION THREAD

**Discussion by newmedicine** [TriliumNext/Trilium#4287] [General]
I can't get my head around the CSS required to show the [+] on hover over a note title in the sidebar even when the title exceeds the width of the note:

https://github.com/zadam/trilium/assets/22443747/7d0b1a50-f067-4806-b1d8-563cea7725ce

This would improve usability a lot as it would mean I don't have to increase the sidebar width to add a child note. 

It would also be good to have a "create child object" button in the note edit form?

Thanks

Nick

**Comment by zerebos** [comment]
You should be able to right click and add a child note in that case

![image](https://github.com/zadam/trilium/assets/6865942/f7ecc448-d430-4e1b-8223-2f85c8be2dc7)

**Reply by newmedicine** [reply]
Fair point, Ctrl+P isn't that much of a hardship. Still feels like a UI irritation though!

**Comment by zadam** [comment]
Fixed in https://github.com/zadam/trilium/commit/3e8f09d30679f449d61441e5bbbb0c82a48699b9
